scripts/modules/cli.py

```diff
@@ -614,7 +635,7 @@ def start_handler(self):
                 del services["opbeans-load-generator"]
 
         compose = dict(
-            version="2.4",
+            version="3.5",
```

Comment:
See my comment in the thread -> https://github.com/elastic/apm-integration-testing/pull/1028/files#r555833169

Reply:
Reverted back to 2.4